build.sbt

```
@@ -9,7 +9,7 @@ ThisBuild / resolvers += "Sonatype OSS Snapshots" at "https://oss.sonatype.org/c
 
 val akkaVersion                = "2.6.18"
 val akkaHttpVersion            = "10.2.7"
-val alpakkaKafkaVersion        = "2.1.1"
+val alpakkaKafkaVersion        = "3.0.0"
 val alpakkaVersion             = "3.0.4"
 val quillJdbcMonixVersion      = "3.7.2"
 val postgresqlJdbcVersion      = "42.2.24"
```

```suggestion
val postgresqlJdbcVersion      = "42.2.25"
```
There is CVE in postgresjdbc https://nvd.nist.gov/vuln/detail/CVE-2022-21724

---

Just rebased and pushed the change